### DISCUSSION TITLE
我对烟味极其敏感

### DISCUSSION THREAD

**Discussion by NEBrightULA** [share121/inter-knot#4879] [灌水]
![1725918282019](https://github.com/user-attachments/assets/8e24297c-f406-4d77-a947-a6a1cc111d12)
如题，哪怕再细微的烟气也能闻到。包括烟草味，烧东西的味道，家里人在家门口关着门点了根烟也能闻到。
特别是过去一星期，附近村子里有人天天半夜烧稻草，甚至在公园里烧。离我家直线距离六七百米，我家开着空调门窗关着，也能闻到那股味道。我和家里不抽烟的说这事，他们都说没闻到。后来以公园火灾隐患为由报了警。
多数时候还不是鼻子闻到的。感受到烟味时，最先反应的是嗓子眼和口腔，嗓子眼会发紧发涩，很难受，口腔里尤其是舌头，也会发涩，还会发麻，特别是舌头两侧，甚至会「尝」到一股烟味特有的味道。令我想起蛇蛇吐信子的样子。
总而言之，几乎相当于一个人形烟雾报警器。不知有没有和咱一样的。

**Comment by QingyunHanyue** [comment]
咱也是这样，不止烟味，还有油漆味，甲醛味等等<(｀^′)>

**Comment by BJ-cmyk** [comment]
欸嘿嘿，尼古丁过敏，抽不了一点😅
